static/js/admin.js

```diff
@@ -959,13 +959,15 @@ function _setup_page() {
         var aliases_info = $("#realm_aliases_modal").find(".aliases_info");
         var data = {
             domain: JSON.stringify($("#new_alias").val()),
+            allow_subdomains: JSON.stringify($("#allow_subdomains").prop("checked")),
         };
 
         channel.post({
             url: "/json/realm/domains",
             data: data,
             success: function () {
                 $("#new_alias").val("");
+                $("#allow_subdomains").prop("checked", false);
```

Comment:
I'm not sure this line makes sense.  Don't we want to put it to the new value?  I'd either re-think this line of code or add a comment explaining why you're doing it.